### ISSUE TITLE
DateTime tolerance cannot be defined as one-sided

### ISSUE THREAD

**Issue by aniketh02** [Seddryck/NBi#317]
I want "- 2days" in my testcase: 
![image](https://user-images.githubusercontent.com/29744147/32143960-8bae2d66-bcd7-11e7-8c47-4b67069fc1d1.png)

Getting the following error. Is it possible to mention - or + value for datetime ?

![image](https://user-images.githubusercontent.com/29744147/32143972-be63e2be-bcd7-11e7-8cd9-88cea4341601.png)

**Comment by Seddryck** [comment]
No, it's not possible at this moment. [One-sided tolerance](http://www.nbi.io/docs/compare-tolerances-roundings/#one-sided-tolerance) is only supported for the type 'numeric'.

Shouldn't be too difficult to add this, just a bit of the same implementation than for NumericComparer and NumericTolerance. If anyone is interested to work on this feel free.
